Ginger/Ginger/BusinessFlowPages_New/NewAutomatePage.xaml.cs

```diff
@@ -91,24 +91,43 @@ private void SetUIControls()
             xBusinessFlowItemComboBox.SelectedIndex = 0;
 
             xAppsAgentsMappingFrame.Content = new ApplicationAgentsMapPage(mContext);
+            mContext.PropertyChanged += Context_PropertyChanged;
             BindEnvsCombo();
             UpdateContext();
         }
 
+        /// <summary>
+        /// This is the context changed handler
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void Context_PropertyChanged(object sender, System.ComponentModel.PropertyChangedEventArgs e)
+        {
+            if (e != null && e.PropertyName == nameof(Activity))
```

Comment:
why we need Context_PropertyChanged in NewAutomatePage? the only item which allowed to change values of Automate Context is NewAutomatePage so why it need to listen to Context changes if it is the one who change it?

Reply:
As discussed changed the code